folly/experimental/test/StringKeyedBenchmark.cpp

```diff
@@ -59,7 +59,7 @@ static const folly::StringPiece lookupPiece{
 #define FOLLY_HAVE_COMPARE_EQUIVALENT 1
 #endif
 
-#if !defined(FOLLY_HAVE_COMPARE_EQUIVALENT) && __GNUC__ >= 5
```

Comment:
@yfeldblum it seems we can just get rid of `FOLLY_HAVE_COMPARE_EQUIVALENT` I think. It will be `1` for Clang 3.4 onward. GCC 5.1 is fine with it too. What's the min Clang version we support?

Reply:
https://github.com/facebook/folly/pull/1157 is a separate MR which removes `FOLLY_HAVE_COMPARE_EQUIVALENT` that you can test internally to make sure it doesn't break anything.